src/Relation/Binary/Domain/Definitions.agda

```diff
@@ -0,0 +1,28 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definitions for domain theory
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Relation.Binary.Domain.Definitions where
+
+open import Data.Product using (∃-syntax; _×_; _,_)
+open import Level using (Level; _⊔_)
+open import Relation.Binary.Bundles using (Poset)
+open import Relation.Binary.Morphism.Structures using (IsOrderHomomorphism)
+
+private
+  variable
+    c o ℓ e o' ℓ' e' ℓ₁ ℓ₂ : Level
+    Ix A B : Set o
+    P : Poset c ℓ e
+
+------------------------------------------------------------------------
+-- Directed families
+------------------------------------------------------------------------
+
+IsSemidirectedFamily : (P : Poset c ℓ₁ ℓ₂) → ∀ {Ix : Set c} → (s : Ix → Poset.Carrier P) → Set _
```

Comment:
This doesn't match the format of definitions elsewhere in the library. 

- In `Module.Definitions` we still just parameterise them by the raw relations/predicates/sets. We don't parameterise the definitions by the bundles themselves.
- The name `Is` is reserved for "Structures", I would just call this `SemiDirected`.
- The name for standard index sets is just `I`, but I think in this case it's not actually being used an index in the type so the standard name `A` or `B` would be better?
- What does the name `s` stand for?